src/test/java/tests/TurboIssueEventTests.java

```diff
@@ -0,0 +1,114 @@
+package tests;
+
+import static org.junit.Assert.assertEquals;
+import github.IssueEventType;
+import github.TurboIssueEvent;
+
+import java.util.ArrayList;
+import java.util.GregorianCalendar;
+import java.util.List;
+
+import org.eclipse.egit.github.core.User;
+import org.junit.Test;
+
+public class TurboIssueEventTests {
+
+    private List<TurboIssueEvent> sampleEvents;
+
+    public TurboIssueEventTests() {
+        sampleEvents = new ArrayList<>();
+
+        sampleEvents.add(
+            new TurboIssueEvent(
+                new User().setLogin("A"), IssueEventType.Labeled,
+                new GregorianCalendar(2015, 1, 1, 1, 1, 0).getTime())
+            .setLabelName("A1"));
+        sampleEvents.add(
+            new TurboIssueEvent(
+                new User().setLogin("A"), IssueEventType.Unlabeled,
+                new GregorianCalendar(2015, 1, 1, 1, 2, 0).getTime())
+            .setLabelName("A2"));
+
+        sampleEvents.add(
+            new TurboIssueEvent(
+                new User().setLogin("B"), IssueEventType.Labeled,
+                new GregorianCalendar(2015, 1, 1, 1, 0, 30).getTime())
+            .setLabelName("B1"));
+        sampleEvents.add(
+            new TurboIssueEvent(
+                new User().setLogin("B"), IssueEventType.Labeled,
+                new GregorianCalendar(2015, 1, 1, 1, 1, 0).getTime())
+            .setLabelName("B2"));
+        sampleEvents.add(
+            new TurboIssueEvent(
+                new User().setLogin("B"), IssueEventType.Unlabeled,
+                new GregorianCalendar(2015, 1, 1, 1, 1, 31).getTime())
+            .setLabelName("B1"));
+
+        sampleEvents.add(
+            new TurboIssueEvent(
+                new User().setLogin("C"), IssueEventType.Labeled,
+                new GregorianCalendar(2015, 1, 1, 2, 30, 15).getTime())
+            .setLabelName("C1"));
+
+
+        sampleEvents.add(
+            new TurboIssueEvent(
+                new User().setLogin("D"), IssueEventType.Unlabeled,
+                new GregorianCalendar(2015, 1, 1, 3, 20, 59).getTime())
+            .setLabelName("D1"));
```

Comment:
Looks like a block of code is being repeated several times. Is it possible to abstract into a small utility method?